evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -300,11 +301,12 @@ def test_make_pipeline_from_components(X_y_binary, logistic_regression_binary_pi
     with pytest.raises(TypeError, match="Every element of `component_instances` must be an instance of ComponentBase"):
         make_pipeline_from_components(['RandomForestClassifier'], problem_type='binary')
 
-    imp = Imputer(numeric_impute_strategy='median')
-    est = RandomForestClassifier()
-    pipeline = make_pipeline_from_components([imp, est], ProblemTypes.BINARY, custom_name='My Pipeline')
-    components_list = pipeline.component_graph
-    assert components_list == [imp, est]
+    imp = Imputer(numeric_impute_strategy='median', random_state=5)
+    est = RandomForestClassifier(random_state=7)
```

Comment:
Why 5 above and 7 here?

Reply:
No strong reason other than I wanted coverage for the case when the components the user passes in components for different random states!

Reply:
Ah gotcha. I was confused that they were different but then reread, got it, they don't need to be the same here, 👍 
 thanks!